src/foam/nanos/boot/NSpec.js

```diff
@@ -275,16 +279,7 @@ foam.CLASS({
       code: function() {
         var service = this.__context__[this.name];
         if ( foam.dao.DAO.isInstance(service) ) {
-          this.__context__.stack.push({
-            class: 'foam.comics.v2.DAOBrowseControllerView',
-            data: service,
-            config: {
-              class: 'foam.comics.v2.DAOControllerConfig',
-              dao: service,
-              createPredicate: foam.mlang.predicate.True,
-              editPredicate: foam.mlang.predicate.True
-            }
-          });
+          this.ctrl.memento.value$.set('admin.data' + foam.nanos.controller.Memento.SEPARATOR + this.name);
```

Comment:
Could you just do this.ctrl.memento.value = 'admin.data' + foam.nanos.controller.Memento.SEPARATOR + this.name;

?